addons/servicegraph/promgen/promgen.go

```diff
@@ -30,13 +31,17 @@ import (
 	"istio.io/istio/addons/servicegraph"
 )
 
-const reqsFmt = "sum(rate(istio_requests_total{reporter=\"server\"}[%s])) by (source_workload, destination_workload, source_app, destination_app)"
-const tcpFmt = "sum(rate(istio_tcp_received_bytes_total{reporter=\"server\"}[%s])) by (source_workload, destination_workload, source_app, destination_app)"
+const reqsFmt = "sum(rate(istio_requests_total{reporter=\"server\"%s}[%s])) by (source_workload, destination_workload, source_app, destination_app)"
```

Comment:
`reporter=\"destination\"`

Reply:
@douglas-reid are there any metric changes that happened so that `reporter=\"destination\"` must used instead of `reporter=\"server\"`?

Reply:
https://github.com/istio/istio/pull/7034

the feedback i received was that `client` and `server` were a bit confusing (especially with inbound/outbound telemetry), so we updated to `source` and `destination` to (hopefully) be clearer.

Reply:
@douglas-reid thanx. Have applied the changes and rebased to current master